### PR TITLE
Document Tails install

### LINKED ISSUE #94: Explicitly document installation and usage in Tails
# Feature Request

## Description

In #91 the docs were overhauled and we make mention of using Tails for safe isolation of Sunder from daily use machines. The new docs include mention of Debian installation steps, and Tails 3.5 is based on Debian Stretch, but we should provide explicit platform support targeting Tails, given the high-security nature of the most proposed use cases for Sunder.

## User Stories

As a user, I want to trust shell commands provided in project documentation not to break my package manager.

As a developer, I want to point users to clear and concise documentation on how to install software, lest we confuse users with package breakage.

### PR DESCRIPTION
**Pull request by emkll**
## Status

Ready for review 

## Description of Changes

Provides documentation of Sunder install in Tails 3.7 with persistence and Internet for initial setup.

Fixes #94 

## Testing

Install Sunder on a Tails volume with persistence following the instructions in this PR.

### CODE REVIEW

File: docs/install.rst [hunk 1]
```diff
@@ -23,4 +23,43 @@ Linux (Debian or derivative)
...
+     cd ~/Persistent
+     mkdir sunder && cd sunder
+     cp ~/Persistent/Tor\ Browser/sunder_0.1.1_amd64.deb
+     cp /tmp/sunder/*.deb .
+     # Install packages and and start Sunder:
+     sudo dpkg -i *.deb
+     # start Sunder
+     sunder
+
+Note that you will need reinstall Sunder each time you boot into Tails (with persistence and Admin options enabled) by opening a terminal and running:
```
Comment (pahakalle) [inline docs/install.rst]: missing a 'to' after "you will need"

**Comment by conorsch** [comment]
Will run through these instructions on hardware and report results. Tails 3.8 was recently released, so that's what I'll use. Will update tested version in docs as appropriate.

**Comment by emkll** [comment]
thanks @pahakalle and @conorsch for your review. I've addressed your comments, this should be ready to re-review.

**Comment by conorsch** [comment]
Gold, thanks for the cleanup, @emkll!

**Pull request merged**